### ISSUE TITLE
Windows environment failed to launch 'portainer' from catalog

### ISSUE THREAD

**Issue by kingsd041** [rancher/rancher#10565]
**Rancher versions:**
rancher/server: 1.6.13-rc3
rancher/agent:

**Docker version: (`docker version`,`docker info` preferred)**
17.06.2-ee-6
**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**
Windows Server 2016 Standard
**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**
VirtualBox
**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
single node rancher
**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**
Windows

**Steps to Reproduce:**
1. Use the windows environment
2. Launch `portainer` stack from catalog
**Results:**

The rancher UI has error messages  `Error (Image [index.docker.io/rancher/portainer-agent:v0.1.0] failed to pull: Pulling from rancher/portainer-agent)
Stack/Service:`

```
PS C:\Users\Administrator> docker pull rancher/portainer-agent:v0.1.0
v0.1.0: Pulling from rancher/portainer-agent
image operating system "linux" cannot be used on this platform
```

**Comment by orangedeng** [comment]
This catalog entry should not be seen or used in windows environment. 
@vincent99 We are using `io.rancher.orchestration.supported` to indicate which orchestration the entry supported. If blank, it means it can be used for all orchestration.
I think that we should not support to use the catalog entry if `io.rancher.orchestration.supported` does not have `windows` in it.

**Comment by deniseschannon** [comment]
@loganhz Is this possible on the environment template as well as the "add Catalog" pages?

**Comment by deniseschannon** [comment]
Available with v1.6.13-rc4

**Comment by sabiodelhielo** [comment]
portainer.io item is now marked "Not Compatible" with the Windows Environment
